packages/addresses/src/public-key.ts

```diff
@@ -21,3 +21,20 @@ export async function getAddressFromPublicKey(publicKey: CryptoKey): Promise<Add
     const publicKeyBytes = await crypto.subtle.exportKey('raw', publicKey);
     return getAddressDecoder().decode(new Uint8Array(publicKeyBytes));
 }
+
+/**
+ * Given an {@link Address}, return a {@link CryptoKey} that can be used to verify signatures.
+ *
+ * @example
+ * ```ts
+ * import { getAddressFromPublicKey } from '@solana/addresses';
+ *
+ * const publicKey = await getPublicKeyFromAddress(address);
+ * ```
+ */
+export async function getPublicKeyFromAddress(address: Address, extractable: boolean = false) {
```

Comment:
I think it'd make sense to make this extractable by default, or (my preference) make it non-configurable and always extractable.

When we create a keypair we always make the public key extractable: https://github.com/anza-xyz/solana-web3.js/blob/997cfe0900e68151ebe79eeccb8fea3c026eac53/packages/keys/src/key-pair.ts#L30-L35

If it's not extractable then you can't get back to the `Address` and that's going to be annoying if you end up passing the `CryptoKey` around your app and need to do that later. I don't think there should ever be an issue with having an extractable public key.